backend/api/services/postServices.py

```diff
@@ -62,6 +62,18 @@ def getPostByAuthorId(request, author_id):
     except:
       return Response(status=status.HTTP_404_NOT_FOUND)
 
+  @staticmethod
+  def deletePostByPostId(request, author_id, post_id):
+    try:
```

Comment:
maybe have a check that the requestor is in fact the creator of this post? If u dont want to implement it now for sure write a comment to remind us that its needed